projects/gossip-service-typescript/src/gossip.ts

```diff
@@ -25,14 +20,18 @@ function handleWSError(address: string, url: string) {
 }
 
 export default class Gossip {
-  localAddress: string = crypto.whoAmI();
+  localAddress: string;
   server: WebSocket.Server;
   clients: Map<string, WebSocket> = new Map();
   listeners: Map<string, any> = new Map();
   peers: any = topologyPeers(topology.peers);
+  node_ip: string;
```

Comment:
Style: `nodeIp`, `nodeName`.

Reply:
You're right. innocent copy-paste. I'll fix